spec/support/page_objects/trainees/index.rb

```diff
@@ -20,7 +20,8 @@ class Index < PageObjects::Base
       element :apply_filters, "input[type='submit'][value='Apply filters']"
 
       element :text_search, "#text_search"
-      element :early_years_checkbox, "#level-early_years"
```

Comment:
I don't think you meant to remove this! It's broken a test.

Reply:
Good spot